### DISCUSSION TITLE
What SQL dev tool have you rolled out for end users?

### DISCUSSION THREAD

**Discussion by kharyptho84** [dbeaver/dbeaver#10999] [General]
We support around 500 analysts that write SQL against our databases. We were using a tool that recently went from free to paid (and at a ridiculous price) that we will no longer support in the coming months. For those of you with lots of SQL writing folks in your different departments, is there a SQL tool that your team supports/recommends to users?

**Comment by p91paul** [comment]
The obvious answer here would be DBeaver, of course! You are asking on the support page of one such tool, and a very good one at that.

**Comment by LonwoLonwo** [comment]
We recommend you DBeaver wholeheartedly
